pyproject.toml

```diff
@@ -8,9 +8,7 @@ name = "invest-demo-plugin"
 # Using setuptools_scm is not required, but we highly recommend it
 dynamic = ["version"]  
 dependencies = [
-  "pygeoprocessing", 
```

Comment:
I don't think it's correct to remove these from `project.dependencies`. This serves the same purpose as `requirements.txt` would, and it should correctly document all direct dependencies of the package. From the [setuptools docs](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#declaring-required-dependency), "This is where a package declares its core dependencies, without which it won’t be able to run."